### ISSUE TITLE
.success is not a function (Bower hasn't the last version)

### ISSUE THREAD

**Issue by LeDiegue** [lynndylanhurley/ng-token-auth#342]
This issue was solved in the last commits, but bower hasn't the last version.. :(

**Comment by ahmadalfy** [comment]
I am having this issue right now.

**Comment by ahmadalfy** [comment]
It seem that the problem is happening to Angular 1.6.0 check the [changelog](https://github.com/angular/angular.js/blob/master/CHANGELOG.md)

>$http:
>  remove deprecated callback methods: success()/error() ([b54a39](https://github.com/angular/angular.js/commit/b54a39e2029005e0572fbd2ac0e8f6a4e5d69014))

This need to be addressed quickly and to switch to `then()`.

**Comment by CJBrew** [comment]
Anyone got a workaround? 
Downgrade Angular?

**Comment by ahmadalfy** [comment]
I resolved that by fixing my Angular dependency to version 1.5.10

**Comment by CJBrew** [comment]
Yes, that's better. Thanks. I guess we need a new release? Though I suppose using git as the source for bower.json should also work...

**Comment by ahmadalfy** [comment]
Yes it should work. I however used `resolution`

```
  "resolutions": {
    "angular": "1.5.10"
  }
```

**Comment by norato** [comment]
@ahmadalfy It works form me too, I also had to downgrade a lot of packages

**Comment by delamorinierejh** [comment]
@ahmadalfy I had to do that too. Is there a timeline for this fix?

**Comment by acovrig** [comment]
I tried installing from git: `bower install --save https://github.com/lynndylanhurley/ng-token-auth.git`
and it still wants to downgrade angular to 1.5.3 (>=1.4 <=1.5.3) while almost everything in my app requires angular#1.6.1 (so I can't accept 1.5.3). Is there an easy way to downgrade other packages to match the angular#1.5.3 requirement?

I created my app w/ `yo angular $appname`, BTW.

**Comment by ahmadalfy** [comment]
@acovrig make sure you install `1.5.10`. ng-token-auth won't work with `1.6.x`

**Comment by Rush** [comment]
Shouldn't changing `.success` to `.then` fix it?

**Comment by deric** [comment]
@booleanbetrayal Would it be possible to do a new release?
